### PR TITLE
Release/1.8.0

### PR DESCRIPTION
**Pull request by jenniferboedker**
1.8.0 (2021-05-07)
------------------

**Added**

* Add qube support (<https://github.com/qbicsoftware/core-utils-lib/pull/39>)

* Add DatasetParser as interface for parsing datasets ``life/qbic/datasets/parsers/DatasetParser.groovy`` (<https://github.com/qbicsoftware/core-utils-lib/pull/49>)

* Add ImagingMetadataValidator for validating json files containing imaging metadata ``life/qbic/utils/ImagingMetadataValidator.groovy``, with test ``life/qbic/utils/ImagingMetadataValidatorSpec.groovy`` (<https://github.com/qbicsoftware/core-utils-lib/pull/48>)

* Introduce life.qbic.utils.BioinformaticAnalysisParser class to parse and validate the filestructure resulting from Nfcore pipeline output (<https://github.com/qbicsoftware/core-utils-lib/pull/51>)

**Fixed**

**Dependencies**

* Upgrade ``life.qbic:groovy:data-model-lib:jar:2.4.0`` -> ``2.7.0``

* Downgrade java 11 -> java 8

* Remove parent-pom and introduce a slim own definition of dependencies.

* Re-introduce ``org.apache.commons.commons-lang3`` dependency with version ``3.7.0``

**Deprecated**

### CODE REVIEW

File: .github/workflows/nexus-publish-osgi.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
```
Comment (jenniferboedker) [inline .github/workflows/nexus-publish-osgi.yml:23]: here its still java 11, is that intended?

**Comment by jenniferboedker** [comment]
I would be happy if someone could check the CL for correctness :)

File: .github/workflows/nexus-publish-osgi.yml (hunk 1 above)
Comment (wow-such-code) [reply .github/workflows/nexus-publish-osgi.yml:23]: @sven1103 I think it is?

Comment (sven1103) [reply .github/workflows/nexus-publish-osgi.yml:23]: wont work, as data model lib is Java 8

**Review by wow-such-code — DISMISSED** [review]
all done

**Review by wow-such-code — APPROVED** [review]
perfect

**Pull request merged**